__tests__/SetsSelectors.test.js

```diff
@@ -1458,11 +1458,11 @@ describe('SetsSelectors', () => {
 
             test('return data for scatter', () => {
                 const expected = [
-                    [
-                    {"title": "c", "velocity": 1.73, "weight": 100}, 
-                    {"title": "d", "velocity": 1.83, "weight": 200}, 
-                    {"title": "k", "velocity": 1.43, "weight": 100}]
-                ];
+                    {"setID": "c", "x": 100, "y": 1.73}, 
+                    {"setID": "d", "x": 200, "y": 1.83}, 
+                    {"setID": "k", "x": 100, "y": 1.43}
+                ]
```

Comment:
- [ ] Go over the regression scenarios so I understand why these values are expected

Reply:
This is identical to the previous chart, just with different keys, the regression scenarios are exactly the same.